Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine as prezto-devel
+LABEL maintainer="h@hlo.mx"
+RUN apk --no-cache update && apk upgrade && \
```

Comment:
Instead of --no-cache the common recommendation is this at the end

```
  && rm -rf /var/cache/apk/*
```

And Why do you do `apk upgrade` ? Since you imply `alpine:latest` in `FROM`.
Because If you didn't do that, it would probably shrink the image size significantly, without you having any flaws I can see.